docs/governance/templates/release_issue.md

```diff
@@ -83,20 +87,21 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] Post to the [agonesdev](https://twitter.com/agonesdev) Twitter account.
 - [ ] Run `git checkout main`.
 - [ ] Then increment the `base_version` in [`build/Makefile`][build-makefile]
-- [ ] Move [helm `tag` value][values] is set to {version}+1-dev
-- [ ] Move the [helm `Chart` version values][chart] is to {version}+1-dev
+- [ ] Move [helm `tag` value][values] is set to {version}+1-dev (update tag field value under image)
+- [ ] Move the [helm `Chart` version values][chart] is to {version}+1-dev (update appVersion and version fields)
 - [ ] Change to the `sdks/nodejs` directory and run the command `npm version {version}+1-dev` to update the package version
 - [ ] Move the [`sdks/csharp/sdk/AgonesSDK.nuspec` and `sdks/csharp/sdk/csharp-sdk.csproj`][csharp] to {version}+1-dev
-- [ ] Update the [`sdks/unity/package.json`][unity] package file's `Version` field to {version}+1-dev
+- [ ] Update the [`sdks/unity/package.json`][unity] package file's `version` field to {version}+1-dev
```

Comment:
Since we're fixing things, can we add a link to `[unity]` down below, so we finally don't have a broken markdown link 😄